src/SpotifyApi.NetCore/LibraryApi.cs

```diff
@@ -0,0 +1,410 @@
+﻿using SpotifyApi.NetCore.Authorization;
+using SpotifyApi.NetCore.Models;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SpotifyApi.NetCore
+{
+    /// <summary>
+    /// An implementation of the API Wrapper for the Spotify Web API Library endpoints.
+    /// </summary>
+    public class LibraryApi : SpotifyWebApi, ILibraryApi
+    {
+        #region constructors
+        public LibraryApi(HttpClient httpClient, IAccessTokenProvider accessTokenProvider) : base(httpClient, accessTokenProvider)
+        {
+        }
+
+        public LibraryApi(HttpClient httpClient, string accessToken) : base(httpClient, accessToken)
+        {
+        }
+
+        public LibraryApi(HttpClient httpClient) : base(httpClient)
+        {
+        }
+        #endregion
+
+        #region CheckUserSavedAlbums
+        /// <summary>
+        /// Check if one or more albums is already saved in the current Spotify user’s ‘Your Music’ library.
+        /// </summary>
+        /// <param name="albumIds">Required. A comma-separated list of the Spotify IDs for the albums. Minimum: 1 ID. Maximum: 50 IDs.</param>
+        /// <returns>bool[] an array of true or false values, in the same order in which the ids were specified.</returns>
+        /// <remarks>
+        /// https://developer.spotify.com/documentation/web-api/reference/library/check-users-saved-albums/
+        /// </remarks>
+        public async Task<bool[]> CheckUserSavedAlbums(
+            string[] albumIds,
+            string accessToken = null
+            ) => await CheckUserSavedAlbums<bool[]>(albumIds, accessToken);
```

Comment:
* Method name should be `CheckUsersSavedAlbums`
* Don't `async` `await` here. No need to unwind the Task. This goes for all non-generic methods that are proxies for the Generic version 🙂